build/Default.ruleset

```diff
@@ -29,6 +29,7 @@
     <Rule Id="CA1054" Action="None" />
     <Rule Id="CA1055" Action="None" />
     <Rule Id="CA1056" Action="None" />
+    <Rule Id="CA1063" Action="Error" />
```

Comment:
❗️ Set this to warning, not error. It will become an error when the build sets the "treat warnings as errors" flag without being so disruptive to workflow.